core/src/test/scala/integration/kafka/admin/ReassignPartitionsIntegrationTest.scala

```diff
@@ -582,7 +598,7 @@ class ReassignPartitionsIntegrationTest extends QuorumTestHarness {
       case (brokerId, rack) =>
         val config = TestUtils.createBrokerConfig(
           nodeId = brokerId,
-          zkConnect = zkConnect,
+          zkConnect = zkConnectOrNull,
```

Comment:
I wonder how hard it would be to convert `createBrokerConfig` to take an Option here instead of a null.

Reply:
I was wondering that too. Looks like nearly 200 usages, so probably better to do it separately.